combinations.rb

```diff
@@ -0,0 +1,11 @@
+def combinations(arr1, arr2)
+	new_arr = []
+	arr1.each do |arr1_e|
+		arr2.each do |arr2_e|
+			new_arr << arr1_e + arr2_e
```

Comment:
Here's just a little convention nitpick:

Ruby files usually use an indentation width of only two spaces :-D